contracts/src/chainlink/cairo/emergency/SequencerUptimeFeed/library.cairo

```diff
@@ -69,7 +69,7 @@ end
 
```

Comment:
About storage_var here we are naming them with `s_` and in aggregator.cairo we aren't using the `s_`but we do add `_`at the and of the name like `config_count_` do we have to rename it or is it ok ?

Reply:
Yup, definitely an inconsistency I'm looking to solve. It's part of a bigger effort in the context of this GH Issue:  https://github.com/smartcontractkit/chainlink-starknet/issues/80

For this PR I'm looking to just get the namespaces renamed.